pkg/capabilities/capabilities.go

```diff
@@ -105,6 +105,28 @@ type CallbackExecutable interface {
 	Execute(ctx context.Context, callback chan<- CapabilityResponse, request CapabilityRequest) error
 }
 
+type Validatable interface {
```

Comment:
Could this be on `BaseCapability`?

Reply:
Yes it should be on `BaseCapability`! But I was making incremental changes first. I didn't want to make a change across all capability types at the same time.

Reply:
Added.